services/codeclimate/codeclimate-analysis-redirector.service.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = [
+  // http://github.com/badges/shields/issues/1387
+  // https://github.com/badges/shields/pull/3320#issuecomment-483795000
+  redirector({
+    name: 'CodeclimateCoverageMaintainabilityRedirect',
+    category: 'coverage',
```

Comment:
```suggestion
    category: 'analysis',
```

Guessing categories don't matter much for redirectors but probably should be `analysis` no?

Reply:
Yea, this category and title were both wrong. Copy pasta!